tests/data/lattice_ibrav0_cell_parameters_int.in

```diff
@@ -0,0 +1,36 @@
+&control
+    outdir = './out_nacl'
+    prefix = 'nacl'
+    calculation = 'scf'
+    tprnfor = .true.
+    tstress = .true.
+    pseudo_dir = './pseudo/'
+ /
+ &system
+    ibrav = 0
+    nat = 8
+    ntyp = 2
+    ecutwfc = 70.0
+ /
+ &electrons
+    diagonalization = 'david'
+    conv_thr = 1.0d-9
+ /
+ATOMIC_SPECIES
+ Na  22.98976928 Na.paw.z_9.ld1.psl.v1.0.0-low.upf
+ Cl  35.453      Cl.pbe-n-rrkjus_psl.1.0.0.UPF
+ATOMIC_POSITIONS crystal
+ Na   0.0000000000000000  0.0000000000000000  0.0000000000000000
+ Na   0.0000000000000000  0.5000000000000000  0.5000000000000000
+ Na   0.5000000000000000  0.0000000000000000  0.5000000000000000
+ Na   0.5000000000000000  0.5000000000000000  0.0000000000000000
+ Cl   0.5000000000000000  0.5000000000000000  0.5000000000000000
+ Cl   0.5000000000000000  0.0000000000000000  0.0000000000000000
+ Cl   0.0000000000000000  0.5000000000000000  0.0000000000000000
+ Cl   0.0000000000000000  0.0000000000000000  0.5000000000000000
+CELL_PARAMETERS angstrom
+ 5.6903014761756712 0 0
+ 0 5.6903014761756712 0
+ 0 0 5.6903014761756712
+K_POINTS automatic
+8 8 8 1 1 1
```

Comment:
Since here we are really just testing the cell parameter parsing, I would suggest to keep this as minimal as possible. This will make it easier to read later on and reduce the size of the parsed reference file.
```suggestion
&control
 calculation = 'scf'
/
ATOMIC_SPECIES
 Na  22.99 Na.upf
ATOMIC_POSITIONS crystal
 Na   0.0  0.0  0.0
CELL_PARAMETERS angstrom
 5.7 0 0
 0 5.7 0
 0 0 5.7
K_POINTS gamma
```

Reply:
thanks 👍